### PR TITLE
NFDiv-3396

### PR DESCRIPTION
**Pull request by BWeeks-JusticeGovUK**
https://tools.hmcts.net/jira/browse/NFDIV-3396

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/CorrectRelationshipDetails.java [hunk 1]
```diff
@@ -18,11 +18,12 @@ public void addTo(PageBuilder pageBuilder) {
             .page("correctPaperCaseDetails", this::midEvent)
             .pageLabel("Correct paper case")
             .label("Label-CorrectYourApplication", "### Your application details")
+            .mandatoryWithLabel(CaseData::getSupplementaryCaseType,
```
Comment (amoriartyMOJ) [inline src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/CorrectRelationshipDetails.java:21]: small picky point but can we add NA to the label?

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ApplicantPrayer.java [hunk 2]
```diff
@@ -53,6 +57,26 @@ public enum EndCivilPartnership implements HasLabel {
         private final String label;
     }
 
+    @Getter
+    @AllArgsConstructor
+    public enum JudicialSeparation implements HasLabel {
+
+        @JsonProperty("judicialSeparation") // Get correct text for the label
```
Comment (amoriartyMOJ) [inline src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ApplicantPrayer.java:64]: Comment left over for correct text for labels. Are they done? If so can we remove the comments

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/CorrectRelationshipDetails.java (hunk 1 above)
Comment (BWeeks-JusticeGovUK) [reply src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/CorrectRelationshipDetails.java:21]: Done

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ApplicantPrayer.java (hunk 2 above)
Comment (BWeeks-JusticeGovUK) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ApplicantPrayer.java:64]: Done

**Review by amoriartyMOJ — APPROVED** [review]
LGTM. 2 Comments but approving anyway. Reminder to update JIRA with QA scenarios if not done yet. Ty!

**Comment by BWeeks-JusticeGovUK** [comment]
> LGTM. 2 Comments but approving anyway. Reminder to update JIRA with QA scenarios if not done yet. Ty!

Already done :)  Thanks!

**Pull request closed without merging**